granary/facebook.py

```diff
@@ -579,6 +580,9 @@ def _create(self, obj, preview=None, include_link=False, ignore_formatting=False
           error_plain='No content text found.',
           error_html='No content text found.')
 
+    if obj.get('displayName') and mf2util.is_name_a_title(unicode(obj.get('displayName')), unicode(obj.get('content'))):
```

Comment:
Eek, I'll fix that in mf2util.

On January 2, 2016 7:25:22 PM PST, Ryan Barrett <notifications@github.com> wrote:
>> @@ -579,6 +580,9 @@ def _create(self, obj, preview=None,
>include_link=False, ignore_formatting=False
>>            error_plain='No content text found.',
>>            error_html='No content text found.')
>>  
>> +    if obj.get('displayName') and
>mf2util.is_name_a_title(unicode(obj.get('displayName')),
>unicode(obj.get('content'))):
>
>ugh, mf2util requires it. ok, thanks.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/snarfed/granary/pull/71/files#r48689708